app/study/practice/page.tsx

```diff
@@ -90,14 +94,6 @@ function PracticePage() {
   }, [isShortAnswerQuestion, currentCardIndex]);
```

Comment:
The `Input` element is conditionally rendered based on `!showAnswer && isShortAnswerQuestion`, which I think causes `inputRef.current` to become null when resetting in `nextQuestion`. A `useEffect` is currently required to handle refocusing, as the element isn’t available immediately. Exploring ways to avoid this dependency while ensuring proper state handling.

Reply:
trying to address this in use-hidden-classname-for-form